package.json

```diff
@@ -19,6 +14,12 @@
     "prepublish": "yarn build:library",
     "start": "react-scripts start"
   },
+  "main": "lib/index.js",
+  "files": [
+    "package.json",
+    "README.md",
+    "lib"
+  ],
```

Comment:
Now using `files` array in `packge.json` instead of `.npmignore` to say what should be installed when library is installed in a project, instead of ignoring what shouldn't. This lets us be explicit about what is installed on users machines so it is as small as possible.